macro/derive/tests/ui_pass/variant_count.rs

```diff
@@ -1,3 +1,4 @@
+#![allow(unused)]
```

Comment:
hm, I guess that since the calculated const value is not bound to a variable it fires the `unused` lint. However, compiler always verifies that the compile time code is valid which still makes this test operational